charts/gardener-extension-admission-gcp/charts/runtime/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,9 @@
+{{- if or (not .Values.global.virtualGarden.enabled) (and (and .Values.global.virtualGarden.enabled .Values.global.virtualGarden.user.name) .Values.global.serviceAccountTokenVolumeProjection.enabled) }}
```

Comment:
This is very hard to read/maintain/understand. Can this be split into multiple `{{ if ... }}` lines? I think there are similar/multiple occurrences of this

Reply:
I split it to multiple lines. It should be easier to read now.